recipes/monetdb/build.sh

```diff
@@ -0,0 +1,18 @@
+CONDA_CFLAGS="-I$CONDA_PREFIX/include"
+CONDA_LDFLAGS="-L$CONDA_PREFIX/lib -Wl,-rpath,$CONDA_PREFIX/lib"
+
+./configure \
+    pcre_CFLAGS="$CONDA_CFLAGS" \
+    pcre_LIBS="$CONDA_LDFLAGS -lpcre" \
+    libxml2_CFLAGS="$CONDA_CFLAGS -I$CONDA_PREFIX/include/libxml2" \
+    libxml2_LIBS="$CONDA_LDFLAGS" \
+    CFLAGS="$CONDA_CFLAGS" \
+    LDFLAGS="$CONDA_LDFLAGS" \
```

Comment:
There are `CFLAGS` and `LDFLAGS` defined in the [`toolchain` package](https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh). It appears that those are being overwritten here. `CONDA_CFLAGS` and `CONDA_LDFLAGS` are used in a few places here in this `build.sh` so I'm not sure if you should incorporate the conda-forge `CFLAGS` and `LDFLAGS` in the `CONDA_CFLAGS` and `CONDA_LDFLAGS` variable defs on lines 1/2 or the `CFLAGS`/`LDFLAGS` variable defs here on lines 9/10. But you should definitely include them in one of those two places.